### DISCUSSION TITLE
edit records and export as Switch save

### DISCUSSION THREAD

**Discussion by eadmaster** [petlyh/MK8SE#1] [Ideas]
i'd like  to move the save i have on the WiiU to the Switch while keeping the stuff i have unlocked and the records.

**Comment by petlyh** [comment]
Thanks for your suggestion.
Editing records is something I might implement in the future.
However, I consider MK8 Deluxe support to be out of scope for this project. You should use an MK8 Deluxe save editor instead.

**Comment by eadmaster** [comment]
i've only found [this old save editor for the switch version](https://gbatemp.net/threads/mario-kart-8-deluxe-save-editor.503543/), but it does not allow you to edit the hiscores/beaten cups.

Also there are some notes here about unlocked tracks:
https://github.com/WerWolv/EdiZon_CheatsConfigsAndScripts/tree/master/Save%20research/Mario%20Kart%208%20Deluxe

**Comment by petlyh** [comment]
> i've only found [this old save editor for the switch version](https://gbatemp.net/threads/mario-kart-8-deluxe-save-editor.503543/), but it does not allow you to edit the hiscores/beaten cups.

That's unfortunate but like I said, I don't have any plans to support MK8 Deluxe at this time. I made this editor mostly for my own use and supporting both MK8 and MK8 Deluxe would require substantial effort, perhaps more than you realize. Besides, I think it would make more sense for an MK8 Deluxe editor to allow importing Wii U saves, rather than for this editor to allow exporting to MK8 Deluxe.

> Also there are some notes here about unlocked tracks:

I'm not sure what you're talking about. Tracks aren't even unlockable in MK8 as far as I know, and that link doesn't contain any information about tracks at all.

**Comment by eadmaster** [comment]
i was referring to this part at the end:
 
> TODO
>
> Right now the current script support edit of stats. The games seems to create more files to unlock tracks or karts.

Btw, i was [missing that in MK8D all tracks and chars are already unlocked from the start](https://www.eurogamer.net/mario-kart-8-deluxe-unlockables-4623), so what remains to edit are the unlocked kart parts and the star-ratings in the GP cups.

**Comment by petlyh** [comment]
> The games seems to create more files to unlock tracks or karts.

That information is wrong. For both MK8 and MK8D, all information about what is unlocked is stored inside `userdata.dat`. Separate files are only created for ghost data and replays.

> what remains to edit are the unlocked kart parts and the star-ratings in the GP cups.

I assume you're talking about MK8D since MK8SE already supports unlocking kart parts. As I've said, this is not an MK8D save editor. That *might* change in the future, but I have no plans for that at this time.

You could ask the devs of save editors for MK8D to implement those features, for example in that GBAtemp thread. Also, those 'notes' you linked seem to be from a Switch save editor that supports MK8D, so you could also create a feature request there.
